src/CodeGen_C.cpp

```diff
@@ -35,6 +35,10 @@ extern "C" unsigned char halide_internal_runtime_header_HalideRuntimeD3D12Comput
 
 namespace {
 
+bool endsWith(const string &str, const string &suffix) {
```

Comment:
I think we already have an ends_with function: https://github.com/halide/Halide/blob/master/src/Util.h#L158